test/proposals/ERC1046/TokenMetadata.test.js

```diff
@@ -0,0 +1,33 @@
+import assertRevert from '../../helpers/assertRevert';
+const ERC20WithMetadata = artifacts.require('ERC20WithMetadataMock');
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .should();
+
+const metadataURI = 'https://example.com';
+const invalidMetadataURI = '';
+
+describe('ERC20WithMetadata', function () {
+  context('invalid metadata', function () {
```

Comment:
The last commit removed the need for testing for invalid metadata.

Reply:
thanks, that's what I get for letting travis do my testing for me 😅